### DISCUSSION TITLE
Where is the MSIX file for PowerShell 7.4.5 ?

### DISCUSSION THREAD

**Discussion by HotCakeX** [PowerShell/PowerShell#24188] [General]
It's not here https://github.com/PowerShell/PowerShell/releases/tag/v7.4.5

**Comment by rhubarb-geek-nz** [comment]
In the [store](https://www.microsoft.com/store/productId/9MZ1SNWT0N5D?ocid=pdpshare)?

**Reply by HotCakeX** [reply]
If you look at the previous release notes, MSIX package was always included.

**Comment by smoonlee** [comment]
Hey @HotCakeX, Did you see you can get the msix package now 

![image](https://github.com/user-attachments/assets/65fe939b-b7e2-4f47-9725-61f532d172d0)

**Reply by HotCakeX** [reply]
Thanks, i see it now, i'll update the links in my codes to the new version.
